### ISSUE TITLE
[BUG] notify-push: Missing oc_ for databse username?

### ISSUE THREAD

**Issue by lieblinger** [nextcloud/all-in-one#4027]
I've deployed a Nextcloud AIO installation on kubernetes and had issues getting the notify-push container running. 

I needed to add manually `oc_` in front of the `POSTGRES_USER` enviroment variable to get the container running.

Shouldn't this line:
https://github.com/nextcloud/all-in-one/blob/6e8fee9d9e28b0f92b7c11faa87fe009b8f58b27/Containers/notify-push/start.sh#L48

look like:
```
export DATABASE_URL="postgres://oc_$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST/$POSTGRES_DB"
```

**Comment by szaimen** [comment]
Oh wow, you are right! Thanks for reporting! :)

**Comment by szaimen** [comment]
This is now fixed with v7.10.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
